### PR TITLE
[TW-828] Doc OpenAPI 2.0 support for multi-file APIs and schema docs

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added OpenAPI 2.0 support for multi-file APIs
- Added OpenAPI 2.0 support automatically generated schema docs

### CODE REVIEW

File: src/pages/docs/publishing-your-api/documenting-your-api.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Generating API documentation"
-updated: 2022-10-04
+updated: 2022-01-04
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/publishing-your-api/documenting-your-api.md:3]: Did you mean to also change the year to `2023`?

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Editing an API definition'
-updated: 2022-12-06
+updated: 2023-02-04
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:3]: Did you mean to also change the month to `01`?

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md]: 🤦🏻

File: src/pages/docs/publishing-your-api/documenting-your-api.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/documenting-your-api.md]: I am definitely not wining the "use the correct year" achievement 🤦🏻

**Review by Himanshu0781 — APPROVED** [review]

**Comment by Himanshu0781** [comment]
LGTM ✅

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**